config/post-install/v0.15.0/dummy.go

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package postinstall is a placeholder that allows us to pull in config files
+// via go mod vendor.
+package postinstall
```

Comment:
I don't understand why this file is necessary. Will these config files be used as a dependency in other modules?

Reply:
If for example, we want operator to import these, if there's no go files, we can't import them.

Reply:
For example, see:
https://github.com/knative/eventing/pull/3098